pkg/webservice/handlers.go

```diff
@@ -670,8 +670,11 @@ func getPartitionInfoDAO(lists map[string]*scheduler.PartitionContext) []*dao.Pa
 		partitionInfo.LastStateTransitionTime = partitionContext.GetStateTime().String()
 
 		capacityInfo := dao.PartitionCapacity{}
-		capacityInfo.Capacity = partitionContext.GetTotalPartitionResource().DAOString()
-		capacityInfo.UsedCapacity = partitionContext.GetAllocatedResource().DAOString()
+		capacity := partitionContext.GetTotalPartitionResource()
```

Comment:
Can you avoid using "capacity" variable and set the value directly to capacityInfo.Capacity? or do you see any other benefit?

Reply:
I set the `DAOString()` to `capacityInfo.Capacity` on 675 below. On 677, I use this calculation to get the abs used capacity. If I don't store the calculation itself in its own variable, I either have to convert the string to `*resource.Resource` or recompute the capacity as an arg to `CalculateAbsUsedCapacity`.